pkg/graveler/ref/manager.go

```diff
@@ -193,15 +194,72 @@ func (m *KVManager) ListTags(ctx context.Context, repositoryID graveler.Reposito
 }
 
 func (m *KVManager) GetCommitByPrefix(ctx context.Context, repositoryID graveler.RepositoryID, prefix graveler.CommitID) (*graveler.Commit, error) {
-	return m.db.GetCommitByPrefix(ctx, repositoryID, prefix)
+	it, err := NewKVOrderedCommitIterator(ctx, m.Store(), repositoryID, false)
```

Comment:
Why do you need an interface to get the store? you can just pass m.store

Reply:
done